### DISCUSSION TITLE
SmartShift beta controls via API

### DISCUSSION THREAD

**Discussion by purcell-lab** [amberelectric/public-api#115] [Ideas]
Any chance we can access the new SmartShift beta controls via the Amber API?

![FB_IMG_1655373508642](https://user-images.githubusercontent.com/79175134/174046329-e32235b1-639a-4eb0-a966-1e0f16478292.jpg)

**Comment by madpilot** [comment]
We are looking in to it.

Technically, yes but we have some contractual hoops to jump through.

**Comment by kcoffau** [comment]
Would love access to trial the API (fully experimental)

I’m running HomeAssisstant which connects to my Victron inverter.

15kw of inverter, 80kw battery, 10kw solar.

I’ve started to build my own logic to manage it, and then @purcell-lab has also given a few handy tips and extensions.

but quite a few amber members are keen to connect their batteries via Ha.

happy to give it a trial (and understand I need to cosune your APi)

**Comment by purcell-lab** [comment]
#165
